lsdcGui.py

```diff
@@ -5269,7 +5289,10 @@ def initCallbacks(self):
       self.omega_pv = PV(daq_utils.motor_dict["omega"] + ".VAL")
       self.omegaTweak_pv = PV(daq_utils.motor_dict["omega"] + ".RLV")
       self.sampyTweak_pv = PV(daq_utils.motor_dict["sampleY"] + ".RLV")
-      self.sampzTweak_pv = PV(daq_utils.motor_dict["sampleZ"] + ".RLV")            
+      if (daq_utils.beamline=="nyx"): 
+        self.sampzTweak_pv = PV(daq_utils.motor_dict["sampleX"] + ".RLV")            
```

Comment:
I thought that because we knew the axes were different, we would have put the Z axis into sampleX in the epx.db file. the config change would have reduced the need for this change.

this could get messy though, we would need to look through the code carefully for all places sample x/y/z are used